### DISCUSSION TITLE
Is it possible to automatically type over a closing parenthesis ")" when it's the next character, instead of inserting a new parenthesis?

### DISCUSSION THREAD

**Discussion by altierawr** [zed-industries/zed#17198] [Feature Requests]
For example, I could be writing code like `func TestMyFunc(t *testing.T)` and my cursor is located between the "T" and the closing parenthesis ")". If I now press ")", usually a closing parenthesis will be inserted. Instead I would like the editor to skip over the closing parenthesis since it's the next character and not insert another one. Is this possible? I'm using vim mode.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
